Source/Core/VideoCommon/CustomTextures.cpp

```diff
@@ -140,10 +140,11 @@ void HiresTexture::Update()
   }
 }
 
-void HiresTexture::Prefetch()
+void CustomTexture::Prefetch()
 {
   Common::SetCurrentThreadName("Prefetcher");
 
+  //TODO Fix this whole thing - Consensus reached at IRC logs MAY-2017
```

Comment:
The logs are extensive for this one, I'm sure but don't know recall exactly ... can I excuse myself until after I get some sleep today.